webhook/resourcesemantics/validation/controller.go

```diff
@@ -35,6 +35,10 @@ import (
 	"knative.dev/pkg/webhook/resourcesemantics"
 )
 
+const (
+	workQueueName = "ValidationWebhook"
+)
```

Comment:
```suggestion
const workQueueName = "ValidationWebhook"
```
The torture never stops (c)